src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -501,11 +502,15 @@ private AlternativeStandfirsts buildAlternativeStandfirsts(XPath xpath, Document
         return builder.build();
     }
 
-    private SortedSet<Identifier> getIdentifiers(XPath xPath, Document attributes, String type, UUID uuid) throws XPathExpressionException {
+    private SortedSet<Identifier> getIdentifiers(XPath xPath, Document value, String type, UUID uuid) throws XPathExpressionException {
         if (!ContentType.Type.DYNAMIC_CONTENT.equals(type)) {
             return ImmutableSortedSet.of(new Identifier(METHODE, uuid.toString()));
         }
-        final Node igUUID = (Node) xPath.evaluate(INTERACTIVE_GRAPHICS_UUID_XPATH, attributes, XPathConstants.NODE);
+        final Node igUUID = (Node) xPath.evaluate(INTERACTIVE_GRAPHICS_UUID_XPATH, value, XPathConstants.NODE);
+        if (igUUID == null) {
+            throw new MethodeContentInvalidException(uuid, "The ig-uuid cannot be null");
```

Comment:
I think it would be better to throw the exception `MethodeContentNotEligibleForPublishException`.